query-engine/connector-test-kit-rs/query-engine-tests/tests/writes/data_types/scalar_lists.rs

```diff
@@ -295,6 +295,21 @@ mod scalar_lists {
         Ok(())
     }
 
+    // Test that Cockroach will not work with enum push
+    #[connector_test(only(Cockroach))]
+    async fn cockroachdb_doesnot_support_enum_push(runner: Runner) -> TestResult<()> {
+        create_row(&runner, r#"{ id: 1 }"#).await?;
+        create_row(&runner, r#"{ id: 2 }"#).await?;
+
+        assert_error!(
+            &runner,
+            r#"mutation { updateOneScalarModel(where: { id: 1 }, data: { enums: { push: A }}) { id }}"#,
+            2009
```

Comment:
It's usually better to include a partial error message to see what you're explicitly expecting from the error.